### ISSUE TITLE
Improvements to Touch Screen Warning

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#20576]
### Is your feature request related to a problem? Please describe.

When running the downloaded launcher, NVDA displays a warning that "touch screen inputs cannot be used because NVDA is not installed.  Touch inputs may not behave as expected while the magnifier is running"

This dialog is very wide, and also I can see it would be intrusive for users running pre-release versions who may not even use a touch screen.

### Describe the solution you'd like

I'd like to propose three fixes for this (I can separate them into other issues if needed).

1) Where the warning comes up at the same time as the license agreement, incorporate it into that screen.
2) Where the warning comes up on its own (eg in a portable copy), put the two sentences on separate lines so the dialog is about the same width as the license agreement / launcher screen.
3) At least when the warning comes up on its own, have a "Don't show this warning again" option.

Screenshot of the warning message as it came up on top of the launcher window in case it helps anyone:

<img width="2527" height="567" alt="Image" src="https://github.com/user-attachments/assets/98ebbefb-5410-46e0-ad80-5608b6dff51c" />

### Additional context

I had to lower the magnification level to 1.5x before I could get the whole message on screen - conversely, I can fit the whole launcher screen in view up to 3.5x magnification.

**Comment by seanbudd** [comment]
Closed by #20612
